cafebabel/core/helpers.py

```diff
@@ -59,3 +59,17 @@ def file_exceeds(file_, size):
     blob = file_.read()
     file_.seek(0)
     return len(blob) > size
+
+
+def current_language():
+    lang_domain = current_app.config.get('DOMAIN_LANGUAGES')
+    domain_lang = {v: k for k, v in lang_domain.items()}
```

Comment:
Given that we only use it the reversed way, why don't declaring it directly with `{'domain': 'lang'}` instead?